examples/chart/teleport-kube-agent/tests/__snapshot__/config_test.yaml.snap

```diff
@@ -1014,6 +1014,81 @@ matches snapshot for tolerations.yaml:
     metadata:
       name: RELEASE-NAME
       namespace: NAMESPACE
+matches snapshot for v10.yaml:
+  1: |
+    apiVersion: v1
+    data:
+      teleport.yaml: |
+        teleport:
```

Comment:
Is teleport happy with no version? Is it going to default to `v1` ?

Reply:
Yeah it defaults to v1 if no version is specified